### PR TITLE
Add Korean translation for `docs/ko/docs/advanced/additional-status-codes.md`

### PR DESCRIPTION
**Pull request by ElliottLarsen**
Add Korean translation for `docs/ko/docs/advanced/additional-status-codes.md`

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5c5b79c3530fb41c628aebb7022670c79ae0527c at: https://649a2fbe7e9b761fe0ba4a5f--fastapi.netlify.app

File: docs/ko/docs/advanced/additional-status-codes.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# 추가 상태 코드
+
+기본적으로 **FastAPI**는 `JSONResponse`를 사용하여 응답을 반환하고, *path operation*에서 반환한 내용을 해당 `JSONResponse`에 대입합니다.
```
Comment (junah201) [inline docs/ko/docs/advanced/additional-status-codes.md:3]:
```suggestion
기본적으로 **FastAPI**는 `JSONResponse`를 사용하여 응답을 반환하고, *경로 동작*에서 반환한 내용을 해당 `JSONResponse`에 대입합니다.
```

File: docs/ko/docs/advanced/additional-status-codes.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+기본 상태 코드나 여러분이 *path operation*에서 설정한 상태 코드를 사용합니다.
```
Comment (junah201) [inline docs/ko/docs/advanced/additional-status-codes.md:5]:
```suggestion
기본 상태 코드나 여러분이 *경로 동작*에서 설정한 상태 코드를 사용합니다.
```

File: docs/ko/docs/advanced/additional-status-codes.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## 추가 상태 코드
+
+여러분이 만약 기본 상태 코드 이외의 추가 상태 코드를 반환하시려면 `JSONResponse`와 같이 `Response`를 직접 반환하시고 추가 상태 코드를 직접 설정하시면 됩니다.
+
+예를 들어 여러분이 아이템 업데이트를 허용하고 성공할 경우 HTTP 상태 코드 200 "OK"를 반환하는 *path operation*을 원한다고 가정해 보겠습니다.
```
Comment (junah201) [inline docs/ko/docs/advanced/additional-status-codes.md:11]:
```suggestion
예를 들어 여러분이 아이템 업데이트를 허용하고 성공할 경우 HTTP 상태 코드 200 "OK"를 반환하는 *경로 동작*을 원한다고 가정해 보겠습니다.
```

**Review by junah201 — COMMENTED** [review]
Hello!

After reviewing the newly translated document, I noticed that the term "path operation" is used without a Korean translation. However, in the existing [tutorial/first-steps](https://github.com/tiangolo/fastapi/blob/master/docs/ko/docs/tutorial/first-steps.md) and [tutorial/dependencies/classes-as-dependencies](https://github.com/tiangolo/fastapi/blob/master/docs/ko/docs/tutorial/dependencies/classes-as-dependencies.md) documents, it is translated as "경로 동작" ("path operation" in Korean). To maintain consistency, it would be better to use the same term.

Therefore, I have translated "path operation" as "경로 동작" to ensure consistency.

Thank you!

---

안녕하세요!

새롭게 번역해주신 문서를 확인해보니 `path operation`이라는 단어가 한국어 번역 없이 그대로 사용되어 있습니다.
하지만 기존에 이미 번역되어 있는 [tutorial/first-steps](https://github.com/tiangolo/fastapi/blob/master/docs/ko/docs/tutorial/first-steps.md)와 [tutorial/dependencies/classes-as-dependencies](https://github.com/tiangolo/fastapi/blob/master/docs/ko/docs/tutorial/dependencies/classes-as-dependencies.md) 문서에 `경로 동작` 이라는 용어로 번역되어 있으므로 일관성을 유지하기 위해 같은 용어를 사용하는 것이 더 좋다고 생각합니다.
따라서 `path operation`을 `경로 동작`으로 번역하여 일치하도록 수정하였습니다.

감사합니다.

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**